### ISSUE TITLE
demtsh branch cannot be read with uproot

### ISSUE THREAD

**Issue by AndrewEdmonds11** [Mu2e/TrkAna#149]
Currently, attempts to read the ```demtsh``` branch with uproot causes the following error:

```
ValueError: 'read beyond' in AwkwardForth runtime: tried to read beyond the end of an input
```

I have opened https://github.com/scikit-hep/uproot5/issues/1221 to better understand the issue

**Comment by AndrewEdmonds11** [comment]
A fix has been found and merged into uproot. It should become available in uproot v5.3.8. Will confirm and close this issue when it is available for testing

**Comment by AndrewEdmonds11** [comment]
Uproot has released v5.3.8 so this should now work
